tests/integration_tests/test_llm.py

```diff
@@ -609,6 +619,108 @@ def test_llm_finetuning_strategies_quantized(tmpdir, csv_filename, finetune_stra
     assert preds
 
 
+@pytest.mark.llm
+@pytest.mark.parametrize(
+    "finetune_strategy,adapter_args,quantization,error_raised",
+    [
+        pytest.param(
+            "lora",
+            {POSTPROCESSOR: {MERGE_ADAPTER_INTO_BASE_MODEL: False}},
+            {"bits": 4},
+            (
+                ImportError,
+                "Using `load_in_8bit=True` requires Accelerate: `pip install accelerate` and the latest version of bitsandbytes `pip install -i https://test.pypi.org/simple/ bitsandbytes` or pip install bitsandbytes` ",  # noqa E501
+            ),
+            id="qlora-4bit-not-merged",
+        ),
+        pytest.param(
+            "lora",
+            {POSTPROCESSOR: {MERGE_ADAPTER_INTO_BASE_MODEL: True, PROGRESSBAR: True}},
+            {"bits": 8},
+            (
+                ImportError,
+                "Using `load_in_8bit=True` requires Accelerate: `pip install accelerate` and the latest version of bitsandbytes `pip install -i https://test.pypi.org/simple/ bitsandbytes` or pip install bitsandbytes` ",  # noqa E501
+            ),
+            id="qlora-8bit-merged",
+        ),
+        pytest.param(
+            "lora",
+            {POSTPROCESSOR: {MERGE_ADAPTER_INTO_BASE_MODEL: False}},
+            {"bits": 8},
+            (
+                ImportError,
+                "Using `load_in_8bit=True` requires Accelerate: `pip install accelerate` and the latest version of bitsandbytes `pip install -i https://test.pypi.org/simple/ bitsandbytes` or pip install bitsandbytes` ",  # noqa E501
+            ),
+            id="qlora-8bit-not-merged",
+        ),
+    ],
+)
+def test_llm_lora_finetuning_merge_and_unload_quantized_accelerate_required(
+    csv_filename, finetune_strategy, adapter_args, quantization, error_raised
+):
+    input_features: List[dict] = [text_feature(name="input", encoder={"type": "passthrough"})]
+    output_features: List[dict] = [text_feature(name="output")]
+
```

Comment:
Hey Alex! What is the purpose of these tests? Quantization based tests will only work once we have GPU tests enabled, so these will be skipped. I also think that once we have GPU tests enabled, one likely won't run into the `using load_in_8bit=True requires Accelerate` error. Okay to leave it in here for now, but was curious what thi test is intended to do

Reply:
@arnavgarg1 This test was simply to avoid commenting out those use cases but just run them and confirm that currently the "NotImportedError" is raised.  Please let me know if it is OK to leave as is for now, or do something else.  Thank you.